### PR TITLE
Bottom Navigation

### PR DESCRIPTION
**Pull request by johnswyoon**
Bottom Nav
- linked to other pages
- highlights when on that page
- gotta decide on FridgeBuddy color scheme

Used the following resources
- https://reactnavigation.org/docs/tab-based-navigation/
- https://icons.expo.fyi/

### CODE REVIEW

**Comment by johnswyoon** [comment]
TODO:
- Decide to use ONLY icon or icon + text
- Decide on colour scheme

File: frontend/app.json
Comment (johnswyoon) [inline frontend/app.json:1]: i connected the React Native app to my Expo account for EAS builds

File: frontend/app.json [hunk 1]
```diff
@@ -25,6 +25,12 @@
     },
     "web": {
       "favicon": "./assets/favicon.png"
-    }
+    },
+    "extra": {
+      "eas": {
```
Comment (johnswyoon) [inline frontend/app.json:30]: i connected the React Native app to my Expo account for EAS builds

**Comment by JacobRintjema** [comment]
l g t m

**Pull request merged**